src/components/WebexVideoSettings/WebexVideoSettings.jsx

```diff
@@ -0,0 +1,65 @@
+import React, {useEffect, useState} from 'react';
+import {
+  Select, SelectOption,
+} from '@momentum-ui/react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import {WEBEX_COMPONENTS_CLASS_PREFIX} from '../../constants';
+import WebexLocalMedia from '../WebexLocalMedia/WebexLocalMedia';
+
+/**
+ *
+ * @returns {object} Returns all of available cameras
+ */
+
+const getAvailableCameras = async (setDevices) => {
+  const cameraDevices = await navigator.mediaDevices.enumerateDevices()
+    .then((cameras) => cameras.filter((camera) => camera.kind === 'videoinput'));
+
+  setDevices([...cameraDevices]);
+};
+
+/**
+ * Webex Video Settings component
+ *
+ * @param {string} props.className  Custom CSS class to apply
+ * @returns {object} JSX of the component
+ */
+export default function WebexVideoSettings({className, meetingID}) {
+  const [devices, setDevices] = useState([]);
```

Comment:
There might be a state where the user has no available camera, which would translate in the same value for `devices`, an empty array. We need to distinguish between "loading the cameras" state and "no cameras available" state.

Let's use `null` as the initial value, and as an idicator that cameras are currently loading.

In the render function, please display a message if `devices && devices.length === 0` (which means no camera is available). See an example message here: https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-231001